layers/meta-balena-jetson/recipes-bsp/tegra-binaries/tegra-nvpmodel_%.bbappend

```diff
@@ -0,0 +1,5 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+# Disable nvpmodel service for Seeed J3010 as it causes boot hangs
+# when trying to set NVMe bandwidth on incompatible configurations
+SYSTEMD_AUTO_ENABLE:${PN}:jetson-orin-nano-seeed-j3010 = "disable"
```

Comment:
Hi @DrakoulisAvonic , the Seeed J3010 supports configurable power modes and it is a public device-type. Disabling this feature means other users would no longer be able to configure their desired power modes.

Are you using a NVME that differs from the one shipped with the Seeed J3010? Could you describe what are the incompatible configurations?